multimodal/src/autogluon/multimodal/utils/data.py

```diff
@@ -503,12 +507,13 @@ def data_to_df(
         if len(missing_columns) > 0:
             # assume no column names are provided and users organize data in the same column order of training data.
```

Comment:
If the test data is also a dataframe with the same headers, then the order of columns doesn't matter since we don't need rename them. However, if users provide pure list data (no header info), that's the best we can do.

Reply:
That makes sense. And if it’s list then it’s more likely to be just one column.